src/content/consul/docs-landing/docs.json

```diff
@@ -0,0 +1,4 @@
+{
+	"pageSubtitle": "Lorem ipsum dolor sit amet.",
+	"marketingContentBlocks": []
```

Comment:
Self TODO 📌 I'm going to fill in this content tomorrow based on what I see [on the .io page](https://www.consul.io/docs).

Reply:
✅ Done in https://github.com/hashicorp/dev-portal/pull/721/commits/31a638cb1c50456a23a9dc112c9860829e506785